Sources/Constants.swift

```diff
@@ -63,6 +63,8 @@ public struct NDT7WebSocketConstants {
 
         /// Discover closer Mlab Server
         public static let url = "https://\(hostname)/\(path)"
+        
+        public static let urlv2 = "https://locate.measurementlab.net/v2/nearest/ndt/ndt7"
```

Comment:
you can follow the same standard that is used for url, so, use the hostname and create a new path with a comment about it